include/SFML/Window/Clipboard.hpp

```diff
@@ -43,27 +43,27 @@ class SFML_WINDOW_API Clipboard
 {
 public:
 
-	////////////////////////////////////////////////////////////
-	/// \brief Get the content of the clipboard as string data
-	///
-	/// This function returns the content of the clipboard
-	/// as a string.
-	///
-	/// \return Current content of the clipboard
-	///
-	////////////////////////////////////////////////////////////
-	static String getString();
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the content of the clipboard as string data
+    ///
+    /// This function returns the content of the clipboard
+    /// as a string.
+    ///
```

Comment:
The documentation should be more detailed, for example what happens if what's inside the clipboard is not text data (ie filename(s), image data, etc.)? Maybe we should handle that?